Sources/PropertyWrappers.swift

```diff
@@ -72,6 +88,7 @@ public final class SwiftyUserDefault<T: DefaultsSerializable> where T.T == T {
     public init(keyPath: KeyPath<DefaultsKeys, DefaultsKey<T>>, options: SwiftyUserDefaultOptions = []) {
```

Comment:
This init should go away, I'd rather us have to specify our adapter everytime if we can't initialize it with a default value. To me that makes more sense to do anyway.

Reply:
Good suggestion. Fixed. Looks much cleaner.